### ISSUE TITLE
Gulp export doesn't create 'pattern-library.json'

### ISSUE THREAD

**Issue by aegirfirefly** [fireflylearning/pattern-library#83]
Running gulp export creates all the files except 'pattern-library.json'.
Running gulp without the parameter creates the file in the export folder.
It might be that running gulp export a second time will also create the file.

**Comment by mwilliamson-firefly** [comment]
@dkhuntrods Looking at `gulpfile.js`, it looks like `exportStatus` is immediately invoked, but presumably it's meant to be invoked as a callback? In other words, should:

```
gulp.task('export', ['export:blocks', 'export:less', 'export:js', 'export:icons', 'export:assets'], exportStatus());
```

be:

```
gulp.task('export', ['export:blocks', 'export:less', 'export:js', 'export:icons', 'export:assets'], exportStatus);
```

? Admittedly, I've only had a quick glance, so could easily be misreading. I'll take a proper look tomorrow since I'm about to head off for the day.

**Comment by mwilliamson-firefly** [comment]
Should be fixed by #84, but do holler if that turns out not to be the case.
